TYPO3.Media/Classes/TYPO3/Media/Controller/AssetController.php

```diff
@@ -295,11 +295,11 @@ public function updateAssetResourceAction(AssetInterface $asset, FlowResource $r
         try {
             $this->assetService->replaceAssetResource($asset, $resource, $options);
         } catch (\Exception $exception) {
-            $this->addFlashMessage('couldNotReplaceAsset', '', Message::SEVERITY_OK, [], 1463472606);
+            $this->addFlashMessage('Could not replace asset', '', Message::SEVERITY_OK, [], 1463472606);
             $this->forwardToReferringRequest();
         }
 
-        $this->addFlashMessage('assetHasBeenReplaced', '', Message::SEVERITY_OK, [htmlspecialchars($asset->getLabel())]);
+        $this->addFlashMessage('Asset "%s" has been replaced.', '', Message::SEVERITY_OK, [htmlspecialchars($asset->getLabel())], 1469083617);
```

Comment:
`htmlspecialchars()` should not be required here (it's the task of the view to escape strings)